README.md

```diff
@@ -39,6 +39,11 @@ More details in the chocolatey page: https://chocolatey.org/packages/headset
 
 ### Linux
 
+#### App Stores:
```

Comment:
is "App Store" the proper term?

Reply:
I'm not sure if it's the proper term. Maybe snap calls itself that, but probably not Flathub.
Maybe, instead of links, we can add a badge to our readme:
https://flathub.org/badges
https://github.com/snapcore/snap-store-badges

I was thinking that those badges might be added to headsetapp.co but, maybe we should just link to the stores there, similar to how we link to the `.deb` and `.rpm` in Github when downloading.

Reply:
👍  with badges. 

yes I already changed the website to just link to here: https://github.com/headsetapp/headset-electron#installation